src/DependencyInjection/ControllerAwareTrait.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * This file is part of Symplify
+ * Copyright (c) 2016 Tomas Votruba (http://tomasvotruba.cz).
+ */
+
+namespace Symplify\ControllerAutowire\DependencyInjection;
+
+trait ControllerAwareTrait
```

Comment:
On second thought, there might be a trait that would combine these. You are right.